docs/reference/api/json-rpc.md

```diff
@@ -98,7 +98,9 @@ Transaction object:
 | `from` | Data, 20&nbsp;bytes | Required | Address of the sender. |
 | `to` | Data, 20&nbsp;bytes | Optional for contract creation | Address of the receiver. `null` if this is a contract creation transaction. |
 | `gas` | Quantity | Optional | Gas provided by the sender. The default is `90000`. |
-| `gasPrice` | Quantity | Optional | Gas price provided by the sender in Wei. The default is `0`. |
+| `gasPrice` | Quantity | Optional | Gas price provided by the sender in Wei. The default is `0`. Used only in non [EIP-1559] transactions. |
+| `maxPriorityFeePerGas` | Quantity | Optional | Maximum fee, in Wei, the sender is willing to pay per gas above the base fee. Used only in [EIP-1559] transactions. |
+| `maxFeePerGas` | Quantity | Optional | Maximum total fee (base fee + priority fee), in Wei, the sender is willing to pay per gas. Used only in [EIP-1559] transactions. |
```

Comment:
For both lines 102 and 103, I think saying "per gas" is either confusing or redundant. "the sender is willing to pay." is enough, because it is referring to the transaction gas fee. I know I need to fix that in Infura docs also.

Reply:
The method name itself is confusing, but anyway...